docs/release-notes/cloud/cloud-2022-02.md

```diff
@@ -0,0 +1,93 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud February 2022.
+included_jira_versions: ['2022.Q1.3', '2022.Q1.4']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/4.4.2
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.2.6
+---
+
+# Cloud February 2022
+
+These release notes are for the Codacy Cloud updates during February 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   It's now possible to click on the organization overview chart to drill down on the information and understand which repositories require attention to improve the organization's code quality. (CY-5580)
+
+    ![Drilling down on the Overall quality chart](../images/cy-5580.png)
```

Comment:
@RaquelHipolito Should we mention something about this being a beta feature? We already have a note on the [documentation page for the Organization Overview](https://docs.codacy.com/organizations/organization-overview/#overall-quality-chart).

Reply:
As it is already documented I don't find it to be very detailed, but you could mention it here, maybe just by a tag.

Reply:
I added `(Beta)` at the end of the release note text.